CalibTracker/SiStripChannelGain/python/SiStripGainsPCLHarvester_cfi.py

```diff
@@ -0,0 +1,12 @@
+import FWCore.ParameterSet.Config as cms
+
+SiStripGainsPCLHarvester = cms.EDAnalyzer(
+    "SiStripGainsPCLHarvester",
+    Record              = cms.untracked.string('SiStripApvGainRcd'),
+    CalibrationLevel    = cms.untracked.int32(0), # 0==APV, 1==Laser, 2==module
+    DQMdir              = cms.untracked.string('AlCaReco/SiStripGains'),
+    calibrationMode     = cms.untracked.string('StdBunch'),
+    minNrEntries        = cms.untracked.double(25),
+    GoodFracForTagProd  = cms.untracked.double(0.98),
```

Comment:
@arunhep 
> if I remember correctly, this number used to be 97% 

At least from the configuration as it is currently in release this doens't appear to be the case.
The parameter is defined as such here: https://github.com/cms-sw/cmssw/blob/CMSSW_9_1_X/CalibTracker/SiStripChannelGain/python/computeGain_cff.py#L31

and is not overridden neither from the `SiStripCalMinBias` specialization: 

https://github.com/cms-sw/cmssw/blob/CMSSW_9_1_X/Calibration/TkAlCaRecoProducers/python/AlcaSiStripGainsHarvester_cfi.py

nor from the `SiStripCalMinBiasAfterAbortGap` one:

https://github.com/cms-sw/cmssw/blob/CMSSW_9_1_X/Calibration/TkAlCaRecoProducers/python/AlcaSiStripGainsAfterAbortGapHarvester_cfi.py

May I ask you what is the source of this 97%?
I must add, that no algorithmic change is intended with this PR.